src/main/java/org/spongepowered/api/text/channel/MessageChannel.java

```diff
@@ -0,0 +1,181 @@
+ * to the members.
+ */
+public interface MessageChannel {
+
+    /**
+     * A channel with no members.
+     */
+    MessageChannel TO_NONE = ImmutableSet::of;
+    /**
+     * A channel with all online players as members.
+     */
+    MessageChannel TO_PLAYERS = () -> ImmutableSet.copyOf(Sponge.getGame().getServer().getOnlinePlayers());
+    /**
+     * A channel with the server console as a member.
+     */
+    MessageChannel TO_CONSOLE = () -> ImmutableSet.of(Sponge.getGame().getServer().getConsole());
+    /**
+     * A channel with all online players, as well as the server console, as members.
+     */
+    MessageChannel TO_ALL = () -> ImmutableSet.<MessageReceiver>builder()
+            .addAll(Sponge.getGame().getServer().getOnlinePlayers())
+            .add(Sponge.getGame().getServer().getConsole())
+            .build();
+
+    /**
+     * Gets a message channel that targets all subjects with the given permission.
+     *
+     * @param permission The permission to target
+     * @return The channel
+     * @see PermissionMessageChannel
+     */
+    static MessageChannel permission(String permission) {
+        return new PermissionMessageChannel(permission);
+    }
+
+    /**
+     * Gets a message channel that targets all subjects contained within the given channels
+     * and applies the message transformations of each channel in order.
+     *
+     * @param channels The channels to combine
+     * @return The channel
+     * @see CombinedMessageChannel
+     */
+    static MessageChannel combined(MessageChannel... channels) {
+        return new CombinedMessageChannel(channels);
+    }
+
+    /**
+     * Gets a message channel that targets all subjects contained within the given channels
+     * and applies the message transformations of each channel in order.
+     *
+     * @param channels The channels to combine
+     * @return The channel
+     * @see CombinedMessageChannel
+     */
+    static MessageChannel combined(Collection<MessageChannel> channels) {
+        return new CombinedMessageChannel(channels);
+    }
+
+    /**
+     * Gets a message channel that targets the given sources.
+     *
+     * @param recipients The recipients
+     * @return The channel
+     * @see FixedMessageChannel
+     */
+    static MessageChannel fixed(MessageReceiver... recipients) {
+        return new FixedMessageChannel(recipients);
+    }
+
+    /**
+     * Gets a message channel that targets the given recipients.
+     *
+     * @param recipients The recipients
+     * @return The channel
+     * @see FixedMessageChannel
+     */
+    static MessageChannel fixed(Collection<? extends MessageReceiver> recipients) {
+        return new FixedMessageChannel(recipients);
+    }
+
+    /**
+     * Broadcast a message to this channel, transforming and sending it to
+     * the members.
+     *
+     * @param original The original text to send
+     */
+    default void broadcast(Text original) {
```

Comment:
What about naming this method `send` or naming the other message `broadcast` as well.
Because having a/no sender does not really make this a/non broadcast at all.
Broadcast is more about the receivers (send to all) and in this matter both methods send to the same receivers.